packagemanifests/managed-api-service/1.12.0/managed-api-service.clusterserviceversion.yaml

```diff
@@ -626,7 +626,6 @@ spec:
   maturity: alpha
   provider:
     name: rhoam
```

Comment:
```suggestion
    name: rhoam
    replaces: managed-api-service.v1.11.0
```

Reply:
Here I basically want to put back the line that release pipeline deleted for some unknown reason.
This shall be addressed in https://issues.redhat.com/browse/MGDAPI-2585